samples/browser/src/main/java/org/mozilla/samples/browser/addons/AddonsFragment.kt

```diff
@@ -139,25 +141,38 @@ class AddonsFragment : Fragment(), AddonsManagerAdapterDelegate {
         }
     }
 
+    private fun showInstallationDialog(addon: Addon) {
+        if (isInstallationInProgress) {
+            return
+        }
+        val addonCollectionProvider = context!!.components.addonCollectionProvider
+        val dialog = AddonInstallationDialogFragment.newInstance(
+            addon = addon,
+            addonCollectionProvider = addonCollectionProvider,
+            onPositiveButtonClicked = { _, allowInPrivateBrowsing ->
```

Comment:
This is related to my other comment. We default to false (which is good), but we don't actually have to call `setAddonAllowedInPrivateBrowsing` if the user doesn't check the box, right?